build/webpack/_development.js

```diff
@@ -41,12 +42,33 @@ export default (webpackConfig) => {
           imports   : ['react'],
           locals    : ['module']
         }
-        loader.query.env.development.plugins[0][1].transforms
-          .push(reactTransformHmr)
+
+        loader.query.env.development.plugins =
+          loader.query.env.development.plugins.map(plugin => {
```

Comment:
If you are referring to the verbosity of it all, I don't see a way to be flexible and succinct. It is a starter-kit so the order of the plugins is going to change ads more are added or removed.